src/Features/Core/Portable/Completion/CompletionRules.cs

```diff
@@ -65,9 +82,32 @@ public virtual bool MatchesFilterText(CompletionItem item, string filterText, Co
                 return false;
             }
 
-            var patternMatcher = this.GetPatternMatcher(_completionService.GetCultureSpecificQuirks(filterText));
+            return GetMatchWithWorkaround(item, filterText) != null;
+        }
+
+        // Hack for Turkish:
+        // start with the culture-specific comparison, and fallback to the invariant one if that fails
+        protected PatternMatch? GetMatchWithWorkaround(CompletionItem item, string filterText)
+        {
+            var patternMatcher = this.GetPatternMatcher(_completionService.GetCultureSpecificQuirks(filterText), CultureInfo.CurrentCulture);
             var match = patternMatcher.GetFirstMatch(_completionService.GetCultureSpecificQuirks(item.FilterText));
-            return match != null;
+
+            if (match != null)
+            {
+                return match;
+            }
+
+            if (CultureInfo.CurrentCulture.Name == "tr-TR")
```

Comment:
Are there actually other cultures that want the behavior of matching their culture or en-US?